src/NavbarMain.js

```diff
@@ -9,14 +9,16 @@ export default function NavbarMain() {
   return (
     <Navbar bg="dark" variant="dark" expand="sm" className='round-top-edges'>
       <Container>
-        <Navbar.Brand to="../">
-          <img
-            src={logo67x40}
-            alt="Partial bingo board with words lingo bingo emblazoned over the top."
-            width={'67px'}
-            height={'40px'}
-          />
-        </Navbar.Brand>
+        <NavLink to="../">
+          <Navbar.Brand>
+            <img
+              src={logo67x40}
+              alt="Partial bingo board with words lingo bingo emblazoned over the top."
```

Comment:
Since this is now a link, could you add to the alt text to indicate it links back to home?